dags/webrender.py

```diff
@@ -37,7 +37,8 @@
             # TODO: remove this after verifying permissions
             RUNFAST="true",
             BUCKET="gs://moz-fx-data-prod-analysis",
-            PROJECT_ID="moz-fx-data-bq-data-science",
+            PROJECT_ID="moz-fx-data-shared-prod",
+            DATASET="telemetry",
```

Comment:
Shouldn't the table be written to `telemetry_derived` instead? We could then create a view in `telemetry` that references this table.